.env.sample

```diff
@@ -0,0 +1,16 @@
+# Database
+DB_USER=wire-poll-app
+DB_PASSWORD=super-secret-wire-pwd
+DB_URL=jdbc:postgresql://db:5432/poll-app
+
+# Init
+SDK_APP_ID=366aca4c-29a5-4641-bddd-672ddd65a320
+SDK_APP_TOKEN=my-api-token
+API_HOST_URL=https://staging-nginz-https.zinfra.io
+CRYPTO_PASSWORD=myDummyPasswordmyDummyPassword01
+
+# SDK
+WIRE_SDK_USER_ID=3adad8bb-4564-4040-b4d0-e4e8156c137a
+WIRE_SDK_EMAIL=marian.kijewski+poll.app@wire.com
+WIRE_SDK_PASSWORD=Aqa123456!
+WIRE_SDK_ENVIRONMENT=staging.zinfra.io
```

Comment:
Even though its a sample, should we still keep these values or just add dummy values, something like:

```
SDK_APP_ID=uuid-value
SDK_APP_TOKEN=my-api-token
API_HOST_URL=https://myhost.dummy
CRYPTO_PASSWORD=myDummyPasswordmyDummyPassword01
```
and etc..

Reply:
Yep, done